modules/fedimint-ln-server/src/lib.rs

```diff
@@ -946,9 +953,10 @@ impl Lightning {
     async fn new(
         cfg: LightningConfig,
         our_peer_id: PeerId,
+        bitcoin_rpc: BitcoinRpcConfig,
         shared: &ServerModuleSharedBitcoin,
     ) -> anyhow::Result<Self> {
-        let btc_rpc = create_bitcoind(&cfg.local.bitcoin_rpc)?;
+        let btc_rpc = create_bitcoind(&bitcoin_rpc)?;
```

Comment:
By always using the new env vars/command line arguments we will break some old setups.

1. Can we avoid that by only ignoring the local config if it's set to a dummy value?
2. Alternatively, how can we guide admins what to put into these (for them) new config options? Maybe we can mention in the argument description that these were previously sourced from `local.json` and if they are unsure what to put there they should look there? Maybe we could even print out the old value for easier migration?